.github/workflows/build.yml

```diff
@@ -0,0 +1,63 @@
+name: Build and upload artifacts
+
+on:
+  push:
+    branches:
+      - main
+      - release/**
+  pull_request:
+    branches:
+      - '*'
+
+env:
+  CACHED_BUILD_PATHS: |
+    ${{ github.workspace }}/plugin-build/build/distributions/*.zip
```

Comment:
we should not cache this

Reply:
Why? This cache is to keep the distribution artifact built in `job_build` available in `job_upload_artifacts`.

Reply:
not sure what to answer since I don't know what `job_build` and `job_upload_artifacts` does, I believe only `actions/upload-artifact@v2` is enough to upload the files and that's it